### DISCUSSION TITLE
What is Free Energy?

### DISCUSSION THREAD

**Discussion by ta-rekt** [josePereiro/TankeFactory#15] [Discussions]
I wrote down your idea @josePereiro on the Boltzmann distribution being the dist that minimizes E-TS. Putting it here to discuss further.

You are a businessman and you want to find the investment strategy that maximizes your return $F$, but you are also in a rush, so that strategy should not be so complex that it takes forever to find it. So you have a cost function $G$ that penalizes time spent. How much do you care about time, versus money? This is parametrized by a number $\beta$ ranging from 0 (I am not in a rush at all, let it take forever) to infinity (I am IN A HURRY).

So, you want to find

$$\arg \max_{\theta} F(\theta) - \beta G(\theta)$$

You run this optimization, and plot the value of your optimal return $F^*$ as a function of your optimal time spent, for different values of $\beta$. You get a curve that looks kind of like this:
[paretos_20240722.pdf](https://github.com/user-attachments/files/16396790/paretos_20240722.pdf)

Now beta happens to be the slope of this curve. It is the tangent, the point at which it is neither advantageous to go left (in the direction of smaller $G$) or right (in the direction of larger $F$). It is an `equilibrium'. You can show this:

$$ \nabla \mathcal{L} = 0 \implies \nabla F^* = \beta \nabla G^*$$

$$ \implies \beta = \frac{\partial F^* }{\partial \theta_i} / \frac{\partial G^* }{\partial \theta_i} = \frac{\partial F*}{\partial G^* } \textcolor{white}{G^*}$$

Now imagine you are the Universe, and you want to find a distribution parametrized by $\theta$ that minimizes average energy $E$ while maximizing entropy $S$. How much do you care if the entropy is really maximized? This is your $\beta = \frac{\partial E}{\partial S}$, which is temperature. Free energy will be the value of this optimum that nature finds, given a fixed `entropic penalty factor'. If $\beta = 0$, the system doesn't care about entropy at all. In this case it will be zero and the system will fall to the minimum energy state, and its distrbution will be $\rho(x) = \delta(x_0)$. If $\beta$ is large, we care a lot about entropy, and we will sacrifice finding a smaller energy just to keep the entropy high. 

So the Boltzmann distribution is not only the one that maximizes entropy at fixed energy, it also minimizes energy at fixed entropy.

**Comment by josePereiro** [comment]
### Tangent

The Maximum Entropy Principle applied on the typical physical problem produce very sharp distributions. That makes all statistics basically the same (ave ~ median ~ mode ~ ...) There is no reason why to peek any of those as the connection between experimental and modelled data... This is a week point in the logic shain from the Hamiltonian till the State Function.
